src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -72,8 +74,14 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
-    def getByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
-      clusterQueryWithLabels.filter { _._1.googleProject === project.string }.filter { _._1.clusterName === name.string }.result map { recs =>
+    def getActiveClusterByName(project: GoogleProject, name: ClusterName): DBIO[Option[Cluster]] = {
+      clusterQueryWithLabels.filter { _._1.googleProject === project.string }.filter { _._1.clusterName === name.string }.filter{_._1.destroyedDate === Timestamp.from(Instant.parse(dummyDate))}.result map { recs =>
```

Comment:
Can you break this out onto multiple lines for readability? Something like:

```
clusterQueryWithLabels
.filter { _._1.googleProject === project.string }
.filter { _._1.clusterName === name.string }
.filter{_._1.destroyedDate === Timestamp.from(Instant.parse(dummyDate))}
.result map { recs =>
```

Reply:
(and/or you can just put all the conditionals in a single filter)

Reply:
+1 for shorter lines